src/main/kotlin/org/elm/ide/docs/ElmDocumentationProvider.kt

```diff
@@ -144,11 +150,7 @@ private fun documentationFor(decl: ElmModuleDeclaration): String? = buildString
     definition {
         i { append("module") }
         append(" ", ids.last().text)
-
-        if (ids.size > 1) {
-            i { append(" declared in ") }
-            ids.dropLast(1).joinTo(this, ".") { it.text }
-        }
+        renderDefinedInQIDs(ids.dropLast(1))
```

Comment:
Rather than display the nested module `Foo.Bar` as `Bar defined in Foo`, I would rather just show it as `Foo.Bar` and get rid of the defined-in clause. I never think about sub-modules independently of the parent module.

And then `renderDefinedInQIDs` can either be inlined or changed so that it takes an upper-case QID parameter instead of a list of psi elements.